### DISCUSSION TITLE
Radarr integration moving files not permitted

### DISCUSSION THREAD

**Discussion by Raasu2** [jsaddiction/TrailerTech#55] [Q&A]
I using Radarr and this script on a debian system. 

After a movie is imported the script is run correctly, trailer is found and downloaded, but script fails to move the faile to the movie folder. Radarr itself has no issues importing movies.  

WARNING - downloaders.downloader - Failed to move MOVIE-trailer.mp4 ERROR: [Errno 1] Operation not permitted: 'path/MOVIE-trailer.mp4'

Running the script from console is working correctly.

**Comment by jsaddiction** [comment]
This honestly sounds like a permissions issue.  The destination folder would need to allow write perms from the application's perspective.  Using docker makes things slightly better because you can force the container to start with specific PUID and PGID.

**Reply by Raasu2** [reply]
Weird. Now i noticed that the trailer is actually being downloaded and moved but it still produces the error. So i think problem is mostly solved for me apart from the error message. 
 
Just some extra info:
The destination folder permissions were set to 'drwxrwxrwx' to rule out permission issue on destination folder. Radarr itself can move files to the destination folder while importing without issues.
